dateparser/search/__init__.py

```diff
@@ -45,7 +45,7 @@ def search_dates(text, languages=None, settings=None, add_detected_language=Fals
          ('on May 6th 2004', datetime.datetime(2004, 5, 6, 0, 0))]
 
 
-        """
+        """ # noqa E501
```

Comment:
In this case, it should also be possible to refactor the code, using multiple lines for the input line. And given that it’s code, so line breaks won’t be added as part of the documentation rendering, I would think it’s important to break that line.

Reply:
Done!